spec/unit/pdk/generate/module_spec.rb

```diff
@@ -24,6 +24,13 @@
   end
 end
 
+shared_context 'allow post generate info to be printed to stdout' do
+  before(:each) do
+    allow($stdout).to receive(:puts).with(a_string_matching(%r{module.*generated at path.*}i))
+    allow($stdout).to receive(:puts).with(a_string_matching(%r{In your new module directory, add classes with the 'pdk new class' command}i))
```

Comment:
shouldn't that be `expect post generate info...`?

Reply:
Yep. New updates should reflect actual testing. lol.